pytest26_extra_requirements.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+
+# Write extra requirements for running pytest with python2.6 here:
+# NOTE: If your pytests need ansible, use the requirements from
+# ansible26_requirements.txt
```

Comment:
Just wondering, does this mean to copy the contents from ansible26_requirements here or does adding `-r ansible26_requirements.txt` work here?

Reply:
See if the comment in the pytest26_extra_requirements.txt in the latest commit answers your question